src/obb_rules/game_progress.cljc

```diff
@@ -0,0 +1,54 @@
+(ns obb-rules.game-progress
```

Comment:
:+1: This is cool, thanks! I looked into the clojure repo and I agree :) I am completely in favor of documenting progress as much as possible, specially when the language provides proper tools to do so.